### ISSUE TITLE
Using lerna version with lerna publish

### ISSUE THREAD

**Issue by erquhart** [lerna/lerna#1762]
## tl;dr
`lerna version --no-push && lerna publish --from-git` doesn't work. Why?

## Expected Behavior

I'm attempting to make changes between `lerna version` and `lerna publish`. I'll spare you the details, but some of our bumps require manual override, which is especially difficult when using conventional commits and independent versioning.

The big question is how `lerna version` and `lerna publish` are supposed to work together in the first place. I would expect to do the following:

1. run `lerna version --no-push`
2. manually make alterations
3. amend Lerna's publish commit with said alterations
4. move tags from old commit to new
5. run `lerna publish --from-git` to actually release

## Current Behavior

The steps described above result in the following message:

```
lerna notice Current HEAD is already released, skipping change detection.
lerna success No changed packages to publish
```

If I also add the `--no-git-tag-version` flag when running `lerna version`, `publish` fails due to uncommited changes in the working tree.

## What now? (formerly "Possible Solutions" 😬)

So here's the thing: I think my expectation to do things between `version` and `publish` makes sense. But the current behavior also make sense. If the current commit is tagged up by Lerna, of course there's nothing to do. If the current working tree is dirty, it isn't clear what those changes represent. All of that makes sense.

What doesn't make sense is the docs decscription of `lerna publish --from-git`:

> This will identify packages tagged by lerna version and publish them to npm.

Can you help me understand what this line from the docs actually means? When I run `lerna version` to tag packages, `lerna publish --from-git` fails _because they're tagged_. I'm certain I'm missing something, just not quite sure what.

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
{
  "lerna": "3.4.3",
  "packages": ["packages/*"],
  "version": "independent",
  "npmClient": "yarn",
  "useWorkspaces": true,
  "loglevel": "success",
  "command": {
    "publish": {
      "allowBranch": "master",
      "conventionalCommits": true,
      "ignoreChanges": ["CHANGELOG.md"]
    }
  }
}

```
</p></details>

**Comment by erquhart** [comment]
cc/ @evocateur

**Comment by evocateur** [comment]
`from-git` is a positional argument, not a flag.

**Comment by erquhart** [comment]
🤦‍♂️👌🏼
